STYLE.md

```diff
@@ -40,25 +40,39 @@ struct Person { name: String }
 
 <!-- dprint-ignore-end -->
 
-if the `Person` struct is not important for your example. Please use this
-sparingly: enclose the code block in `<!-- dprint-ignore-start -->` and
-`<!-- dprint-ignore-end -->` to suppress warnings about the formatting.
+Please use this sparingly: enclose the code block in
+`<!-- dprint-ignore-start -->` and `<!-- dprint-ignore-end -->` to suppress the
+automatic formatting.
+
+### Language and Tone
+
+The courses are written in American English, so write "initialize", not
+"initialise".
+
+Use an informal, friendly, and concise tone. Remember that the courses are meant
+to be taught by experienced programmer to other experienced programmers. We
```

Comment:
```suggestion
to be taught by experienced an programmer to other experienced programmers. We
```

Reply:
I'll go with this instead:

```suggestion
to be taught by an experienced programmer to other experienced programmers. We
```

:smile: